src/main/java/com/openmodloader/loader/OmlBootstrap.java

```diff
@@ -0,0 +1,82 @@
+package com.openmodloader.loader;
+
+import com.openmodloader.api.IOmlContext;
+import com.openmodloader.api.loader.IModReporter;
+import com.openmodloader.api.mod.Mod;
+import com.openmodloader.api.mod.ModMetadata;
+import com.openmodloader.api.mod.config.IModConfig;
+import com.openmodloader.api.mod.config.IModConfigurator;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public final class OmlBootstrap {
```

Comment:
Perhaps just Bootstrap?

Reply:
Mainly avoided this because of `Bootstrap` from MCP, but realised this isn't MCP. :P
Still, maybe it's worth being more specific?